operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/ConfigurationService.java

```diff
@@ -474,7 +474,8 @@ default boolean previousAnnotationForDependentResourcesEventFiltering() {
    *
    * @return a Set of resource classes where the previous version annotation won't be used.
    */
-  default Set<Class<? extends HasMetadata>> previousAnnotationUsageBlocklist() {
+  default Set<Class<? extends HasMetadata>>
+      previousAnnotationForDependentResourcesEventFilteringBlocklist() {
```

Comment:
I think this is fine to have also with shorter form and keep it consistent. (withPreviousAnnotationForDependentResourcesBlocklist)